src/zenml/cli/served_models.py

```diff
@@ -0,0 +1,238 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+import uuid
+
+import click
+
+from zenml.cli.cli import cli
+from zenml.cli.utils import (
+    declare,
+    error,
+    pretty_print_model_deployer,
+    print_served_model_configuration,
+    warning,
+)
+from zenml.enums import StackComponentType
+from zenml.model_deployers import BaseModelDeployer
+from zenml.repository import Repository
+
+
+@cli.group()
+@click.pass_context
+def served_models(ctx: click.Context) -> None:
+    """Served models that are deployed using the active model deployer in the
+    stack
+
+    Args:
+        ctx: Click context to pass to all sub-commands
+    """
+    ctx.obj = Repository().active_stack.components.get(
+        StackComponentType.MODEL_DEPLOYER, None
+    )
+    if ctx.obj is None:
+        error(
+            "No active model deployer found. Please add a model_deployer to "
+            "your stack."
+        )
+
+
+@served_models.command("list")
+@click.pass_obj
+def list_models(model_deployer: "BaseModelDeployer") -> None:
+    """Get a list of all served models within the model-deployer stack
+    component.
+    Args:
+        model_deployer: Stack component that implements the interface to the
+            underlying model deployer engine
+    """
+    pretty_print_model_deployer(model_deployer.find_model_server())
+
+
+@served_models.command("describe")
+@click.argument("served_model_uuid", type=click.STRING)
+@click.pass_obj
+def describe_model(
+    model_deployer: "BaseModelDeployer", served_model_uuid: str
+) -> None:
+    """Describe a specified served model.
+
+    Args:
+        model_deployer: Stack component that implements the interface to the
+            underlying model deployer engine
+        served_model_uuid: UUID of a served model
+    """
+
+    served_model = model_deployer.find_model_server(
+        service_uuid=uuid.UUID(served_model_uuid)
+    )[0]
```

Comment:
This linebreak triggers me a bit, might have been my fault 
```suggestion
    served_model = model_deployer.find_model_server(
        service_uuid=uuid.UUID(served_model_uuid))[0]
```